1-tokenizer.js

```diff
@@ -0,0 +1,96 @@
+// 1-tokenizer.js
+
+//tokenの作成
+export function tokenizer(input) {
+  let current = 0; //コードのcurrentの位置
+  let tokens = []; 
+
+  while (current < input.length) {
+    let char = input[current]; //inputのcurrentの文字
+
+    if (char === '(') {
+      tokens.push({
+        type: 'paren',
+        value: '(',
+      });
+      current++;
+      continue;
+    }
+
+    if (char === ')') {
+      tokens.push({
+        type: 'paren',
+        value: ')',
+      });
+      current++;
+      continue;
+    }
+
+    //空白のcheck　トークンとして格納する必要はない
+    let WHITESPACE = /\s/;
```

Comment:
nit

I personally think these (most probably immutable) variables such as `WHITESPACE`, `NUMBERS` and `LETTERS`  can/should be declared using `const` at the top level to indicate that they're not gonna change

Reply:
🙆🏻‍♀️